cmd/vic-machine/configure/configure.go

```diff
@@ -161,7 +161,7 @@ func (c *Configure) processParams(op trace.Operation) error {
 // copyChangedConf takes the mostly-empty new config and copies it to the old one. NOTE: o gets installed on the VCH, not n
 // Currently we cannot automatically override old configuration with any difference in the new configuration, because some options are set during the VCH
 // Creation process, for example, image store path, volume store path, network slot id, etc. So we'll copy changes based on user input
-func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n *config.VirtualContainerHostConfigSpec, clic *cli.Context) {
+func (c *Configure) CopyChangedConf(o *config.VirtualContainerHostConfigSpec, n *config.VirtualContainerHostConfigSpec, grantPerms bool) {
```

Comment:
nit: Let's call the new `bool` something a bit clearer like `grantPermsIsSet`. (The current name makes it seem like it holds the _value_ of grant perms, not whether the value has been supplied.)